Fix local setup of UCI

tushar5526 opened the pull request:
Related to `fix` in #23 

- Add UCI-admin to docker-compose and fix networking.
- Add missing vars to .env

ChakshuGautam commented on docker-compose.yml:
```diff
@@ -195,40 +178,36 @@ services:
-              connect:
-                - name: "connect"
-                  url: "http://connect:8083"
+  # akhq:
```
Remove this. Let's create a new docker-compose.debug.yaml that has all these things.

ChakshuGautam commented on docker-compose.yml:
```diff
@@ -283,12 +254,9 @@ services:
     env_file: .env
-    networks:
-      - cassandra
     healthcheck:
```
Consider removing these since they are not getting utilized anywhere. Or change this to the new aysnc ones.

tushar5526 replied:
Removed healthchecks from compose as we already have /health

tushar5526 commented on .env:
```diff
@@ -179,9 +183,9 @@ CDAC_BASE_URL=http://cdac:9095
 
 #Vault Credentials
-VAULT_FUSION_AUTH_URL=http://auth.samagra.io:9011
+VAULT_FUSION_AUTH_URL=http://fusionauth:9011
```
We will have to adjust the env vars name in the service's code as well. Currently, the plan is to provide a working installation way to adopters and then fix pick everything else as enhancements. 

https://github.com/samagra-comms/docker-deploy/issues/23 

Above comment can go under fixes if needed.

tushar5526 commented on .env:
```diff
@@ -179,9 +183,9 @@ CDAC_BASE_URL=http://cdac:9095
 VAULT_FUSION_AUTH_TOKEN=token
-VAULT_SERVICE_URL=http://143.110.183.73:3002
+VAULT_SERVICE_URL=http://uci-api-service:3002
 VAULT_SERVICE_TOKEN=token
```
@pankajjangid05 can you take a look, please ?

pankajjangid05 replied:
Hey @tushar5526 @ChakshuGautam 

This is for get vault credentials. 

https://github.com/samagra-comms/utils/blob/release-v2.0.0/src/main/java/com/uci/utils/BotService.java#L410
https://github.com/samagra-comms/utils/blob/release-v2.0.0/src/main/java/com/uci/utils/BotService.java#L414

tushar5526 replied:
This will not be a part of this PR and added to enhancements.

tushar5526 commented on .env:
```diff
@@ -192,3 +196,40 @@ TELEMETRY_EVENT_PDATA_ID=dev.uci.samagra
+NG_APP_conversationAdapterId=''
+
+#Doubtnut config
+DOUBTNUT_BASE_URL=https://micro.doubtnut.com/secure/ask/tara
```
https://github.com/samagra-comms/transformer/blob/master/src/main/resources/application.properties - this has to be fixed first at the transformer service. We can add a comment stating the variables are optional and will be fixed.

pankajjangid05 replied:
Hey @ChakshuGautam 

I have added default values in transfromer for Doubtnut and NL env. variables, so that we can remove from docker-deploy .env
and I have created a PR for it can you please review it : https://github.com/samagra-comms/transformer/pull/64

tushar5526 commented:
Dependent on https://github.com/samagra-comms/uci-apis/issues/61

tushar5526 commented:
@ChakshuGautam I have added a new `manual-setup.sh` and removed some steps of UCI setup as we have dockerized some services of the project. Moreover, starting all services at once, with `restart:always` policy.
